pkg/controller/daemon/controller.go

```diff
@@ -79,7 +88,7 @@ type DaemonSetsController struct {
 	queue *workqueue.Type
 }
 
-func NewDaemonSetsController(kubeClient client.Interface, resyncPeriod controller.ResyncPeriodFunc) *DaemonSetsController {
+func NewDaemonSetsController(kubeClient client.Interface, resyncPeriod controller.ResyncPeriodFunc, burstReplicas int) *DaemonSetsController {
```

Comment:
please don't add this as an argument to the constructor until it's actually configurable.